specification/columns/servicesubcategory.md

```diff
@@ -0,0 +1,126 @@
+# Service Subcategory
+
+The Service Subcategory is the second-highest-level classification of a [*service*](#glossary:service) based on its core function.  The Service Subcategory (in conjunction with the Service Category) is commonly used for scenarios like analyzing costs across providers and tracking the migration of workloads across fundamentally different architectures.  The Service Subcategory is a child of the Service Category, and each Service Subcategory value must have one and only one Service Category parent. Though a given *service* can have multiple purposes, each *service* should have one and only one Subcategory that best aligns with its primary purpose. 
+
+The ServiceSubcategory column MUST be present in a FOCUS dataset and MUST NOT be null. This column is of type String and MUST be one of the allowed values.
+
+## Column ID
+
+ServiceSubcategory
+
+## Display Name
+
+Service Subcategory
+
+## Description
+
+Second-highest-level classification of a *service* based on its core function.
+
+## Content Constraints
+
+| Constraint      | Value          |
+| :-------------- | :------------- |
+| Column type     | Dimension      |
+| Feature level   | Mandatory      |
+| Allows nulls    | False          |
+| Data type       | String         |
+| Value format    | Allowed Values |
+
+Allowed values:
+
+| Service Category          | Service Subcategory               | Service Subcategory Description                                                                                                                                                                                                            |
```

Comment:
@salpaysenturus, there are long tables on this PR; I am volunteering to convert these tables to HTML if you think this will help.

Reply:
@jpradocueva This is why I asked about tables specifically in the editorial guidelines during the member call yesterday.  If you feel that long tables should be in HTML instead of markdown as a matter of course, it would be helpful for that documentation to state as such.

As it stands, I plan to keep the supporting spreadsheet and the PR in sync by making changes in the spreadsheet and using a converter for the markdown.  I can easily switch that to HTML -- but I'd only prefer to do that if it's a standard we're looking to apply across the entire spec.

Reply:
Resolving this conversation and leaving it markdown.